### PR TITLE
[FX-4132] Update Groups documentation re: removal of Associated Asset…

### PR DESCRIPTION
**Pull request by mtsuji-cobalt**
### Removed

References to the Associated Assets field during group creation

### Updated

Groups form screenshot without the Associated Assets field

## Preview This Change

To see how this change looks in production, scroll down to **Deploy Preview**. Select the link that looks like `https://deploy-preview-<num>--cobalt-docs.netlify.app/`

## Variables

Help us support a “Write once, publish everywhere” single source of truth. If you see a line that looks like:

`{{% asset-categories %}}`

You’ve found a [shortcode](https://gohugo.io/content-management/shortcodes/) that we include in multiple documents.

You’ll find the content of the shortcode in the following directory:

https://github.com/cobalthq/cobalt-product-public-docs/tree/main/layouts/shortcodes

That shortcode has the same base name as what you see in the PR, such as `asset-categories.html`.

## Checklist for PR Author

[ ] Did you check for broken links and alt text?

Be sure to check for broken links and Alt Text issues. We have a partially automated process,
as described in this section of our repository README:
[Test Links and Alt Attributes](https://github.com/cobalthq/cobalt-product-public-docs/blob/main/README.md#test-links-and-alt-attributes).

### CODE REVIEW

File: content/en/Platform Deep Dive/Collaboration/groups.md [hunk 1]
```diff
@@ -16,19 +16,22 @@ Watch a demo video going over the Groups feature:
...
   - **Group Name**: Set up a descriptive name to easily identify the group.
   - **Description**: Add information to further describe the group.
-  - **Associated Assets**: Select from a list of all assets within your organization (Note: Assets already associated with another group will not appear in this list).
   - **Members**: Select from a list of all Organization Members within your organization. These users will have exclusive access to the group’s associated assets, and their pentests and findings.
     - Organization Owners and Cobalt Staff will not appear in this list but will still have access to all assets.
     - [Pentest Team Members](/platform-deep-dive/collaboration/user-roles/#pentest-team-member) that are not part of your organization can still be [manually added](/platform-deep-dive/collaboration/manage-collaborators/#add-a-pentest-team-member) to each pentest.
 - Select **Create Group**.
 
+{{% alert title="Note" color="primary" %}}
+Assets can be [individually assigned](/collaboration/groups/#assigning-a-group-to-an-asset) to a group from the Create or Edit Asset forms.
```
Comment (mtsuji-cobalt) [inline content/en/Platform Deep Dive/Collaboration/groups.md:32]: I don't know why the line breaks here.

Comment (mohitkyadav) [reply content/en/Platform Deep Dive/Collaboration/groups.md:32]: It is still one line, it wraps because there's not enough space.

**Review by mohitkyadav — APPROVED** [review]

File: content/en/Platform Deep Dive/Collaboration/groups.md (hunk 1 above)
Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Collaboration/groups.md:32]: Screenshot 2024-05-15 at 07 49 37
Hmm ok. It still looks like I have space, though.

Comment (mohitkyadav) [reply content/en/Platform Deep Dive/Collaboration/groups.md:32]: Ohh I thought you meant in the PR preview 😅 . For some reason I see 2 links in the rendered HTML. 
Screenshot 2024-05-15 at 17 35 40

Do we know if there are other examples of links inside a note 🤔?

Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Collaboration/groups.md]: Screenshot 2024-05-15 at 08 11 56

there's an example in `platform-deep-dive/collaboration/manage-collaborators/#remove-a-pentest-team-member`

File: content/en/Platform Deep Dive/Collaboration/groups.md [hunk 2]
```diff
@@ -16,19 +16,22 @@ Watch a demo video going over the Groups feature:
...
   - **Description**: Add information to further describe the group.
-  - **Associated Assets**: Select from a list of all assets within your organization (Note: Assets already associated with another group will not appear in this list).
   - **Members**: Select from a list of all Organization Members within your organization. These users will have exclusive access to the group’s associated assets, and their pentests and findings.
     - Organization Owners and Cobalt Staff will not appear in this list but will still have access to all assets.
     - [Pentest Team Members](/platform-deep-dive/collaboration/user-roles/#pentest-team-member) that are not part of your organization can still be [manually added](/platform-deep-dive/collaboration/manage-collaborators/#add-a-pentest-team-member) to each pentest.
 - Select **Create Group**.
 
+{{% alert title="Note" color="primary" %}}
+Assets can be [individually assigned](/platform-deep-dive/collaboration/groups/#assigning-a-group-to-an-asset) to a group from the Create or Edit Asset forms.
+{{% /alert %}}
```
[the remark this answers is not in the document]
Comment (mtsuji-cobalt) [reply content/en/Platform Deep Dive/Collaboration/groups.md]: That worked, thanks :)

Screenshot 2024-05-15 at 08 18 15

**Review by TaylorBriggs — APPROVED** [review]

**Pull request merged**